extra/openai/gui/gui.factor

```diff
@@ -0,0 +1,100 @@
+! Copyright (C) 2023 Dave Carlton.
+! See https://factorcode.org/license.txt for BSD license.
+USING: accessors assocs colors fonts io io.encodings.utf8 io.files
+kernel math namespaces openai sequences ui ui.commands ui.gadgets
+ui.gadgets.borders ui.gadgets.editors ui.gadgets.labels
+ui.gadgets.scrollers ui.gadgets.toolbar ui.gadgets.tracks ui.gestures
+ui.pens.solid ui.text ui.tools.common ui.tools.listener urls
+webbrowser wrap.strings ;
+
+IN: openai.openai-gui
+
+INITIALIZED-SYMBOL: OPENAI-KEY-PATH [ "~/.config/configstore/openai-key" ]
```

Comment:
what program uses ``~/.config/configstore``? is this a convention we should adopt more, or is this particular to your setup?

Reply:
Seems to be a common place to place things in the Mac (using brew and friends) and Linux world. It could be anywhere you want the key as OpenAI will deactivate your key immediately if it sees a key in a commit.